tests/layout/TabContainer.html

```diff
@@ -362,8 +362,12 @@
 							cpResizes++;
 						}, true);
 
+						t.is(true, domClass.contains(cp.domNode, 'dijitTabPane'), 'children have correct CSS class');
+
```

Comment:
As a minor thing, it's preferable to call `t.t(...)` rather than `t.is(true, ...)` here.